<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectangleForm</class>
 <widget class="QWidget" name="RectangleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rectangle</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="frame" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>s&amp;ize</string>
        </property>
        <property name="buddy">
         <cstring>width</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="radiusX">
        <property name="placeholderText">
         <string comment="corner radius" extracomment="placeholder">width</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="radiusY">
        <property name="placeholderText">
         <string comment="corner radius" extracomment="placeholder">height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;position</string>
        </property>
        <property name="buddy">
         <cstring>anchorX</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLineEdit" name="rotation">
        <property name="placeholderText">
         <string extracomment="placeholder">rotation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="height">
        <property name="placeholderText">
         <string extracomment="placeholder">height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="anchorY">
        <property name="placeholderText">
         <string extracomment="placeholder">y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="anchorX">
        <property name="placeholderText">
         <string extracomment="placeholder">x</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="width">
        <property name="placeholderText">
         <string extracomment="placeholder">width</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;corner radius</string>
        </property>
        <property name="buddy">
         <cstring>radiusX</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;rotation</string>
        </property>
        <property name="buddy">
         <cstring>rotation</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="styleFrame" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="stroke">
        <property name="text">
         <string>&amp;stroke</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="strokeWidth">
        <property name="placeholderText">
         <string extracomment="placeholder">stroke width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fill">
        <property name="text">
         <string>&amp;fill</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="strokeColor">
        <property name="toolTip">
         <string extracomment="button tooltip">select stroke color</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="fillColor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="button tooltip">select fill color</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>na&amp;me</string>
     </property>
     <property name="buddy">
      <cstring>name</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="anchorGroup">
     <property name="title">
      <string>&amp;anchor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="TopLeft">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="Left">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="TopRight">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="Top">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="BottomRight">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="Right">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="BottomLeft">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="Center">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="Bottom">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">anchorButtons</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="buttonFrame" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="text">
         <string>&amp;New</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>&amp;Delete</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>anchorX</tabstop>
  <tabstop>anchorY</tabstop>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>rotation</tabstop>
  <tabstop>radiusX</tabstop>
  <tabstop>radiusY</tabstop>
  <tabstop>fill</tabstop>
  <tabstop>fillColor</tabstop>
  <tabstop>stroke</tabstop>
  <tabstop>strokeColor</tabstop>
  <tabstop>strokeWidth</tabstop>
  <tabstop>name</tabstop>
  <tabstop>TopLeft</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fill</sender>
   <signal>toggled(bool)</signal>
   <receiver>fillColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stroke</sender>
   <signal>toggled(bool)</signal>
   <receiver>strokeWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stroke</sender>
   <signal>toggled(bool)</signal>
   <receiver>strokeColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="anchorButtons"/>
 </buttongroups>
</ui>
